lib/Frontend/Frontend.cpp

```diff
@@ -425,7 +425,21 @@ shouldImplicityImportSwiftOnoneSupportModule(CompilerInvocation &Invocation) {
   return Invocation.getFrontendOptions().isCreatingSIL();
 }
 
+void CompilerInstance::performParseAndResolveImportsOnly() {
+  performSemaUpTo(SourceFile::NameBound);
```

Comment:
I'm personally not sure we need these helper functions, and think we could just add an argument to `performSema` (and deprecate `performParseOnly` later). But either way is okay with me.